### DISCUSSION TITLE
Αναπλήρωση συνάντησης 1ης εβδομάδας

### DISCUSSION THREAD

**Discussion by epidrome** [upatras-hci/iv#25] [Polls]
Αναπλήρωση 1ης εβδομάδας

**Comment by akimo13** [comment]
Καλημέρα κ. Χωριανόπουλε (@epidrome ), η αναπλήρωση του μαθήματος θα γίνει σήμερα στις 11:00 π.μ. ; Σύμφωνα με τα αποτελέσματα του poll;

**Reply by epidrome** [reply]
ναι, σήμερα στις 11πμ!
